plugins/premium/comments/comments_end-user_docs.md

```diff
@@ -0,0 +1,101 @@
+---
+layout: default
+title: Using TinyMCE Comments
+title_nav: Using Comments
+description: How to add, edit, resolve, and remove comments in TinyMCE
+keywords: comments commenting tinycomments
+---
+
+{% assign pluginname = "Comments" %}
+{% assign plugincode = "comments" %}
+
+I'm trying to:
+
+- [Add a comment](#addacomment).
+- [Edit a comment](#editacomment).
+- [Delete a comment](#deleteacomment).
+- [Delete a comment thread (conversation)](#deleteacommentthreadconversation).
+- [Resolve a comment thread (conversation)](#resolveacommentthreadconversation).
+- [Show or view a comment](#showorviewacomment).
+- [Delete all comment threads](#deleteallcommentthreads).
+
+## Add a comment
+
+1. Select the text from the desired location in the editor body.
+1. From the navigation menu, choose **Insert**-> **Add Comment** or click on the **Add comment** ![Add comment]({{site.baseurl}}/images/icons/comment-add.svg) toolbar button to add the comment.
+1. The Comment dialog box appears in the sidebar of the editor instance.
+1. Type the comment in the box displaying "_Say something…_" suggested text.
+1. Press **Clear** to delete or **Save** to store the input comment.
```

Comment:
```suggestion
1. Press **Clear** to discard or **Save** to store the input comment.
```
`delete` is a little confusing and using `discard` also keeps things consistent with the edit procedure below.